lib/components/AlbumScreen/preset_chip.dart

```diff
@@ -54,24 +55,21 @@ class _PresetChipsState extends State<PresetChips> {
     offset = min(max(0, offset),
         widget.chipWidth * (widget.values.length) - maxWidth - _spacing);
 
-    _controller.animateTo(
-      offset,
-      duration: Duration(milliseconds: 350),
-      curve: Curves.easeOutCubic,
-    );
+    if (_controller == null) {
+      _controller = ScrollController(initialScrollOffset: offset);
+    } else {
+      _controller!.animateTo(
+        offset,
+        duration: const Duration(milliseconds: 500),
```

Comment:
Any particular reason why you changed the duration? Feels a bit slow now with the cubic curve.

Reply:
I slowed this down to make things a bit easier to see during testing and just forgot to revert it.